plugin-build/build.gradle.kts

```diff
@@ -0,0 +1,101 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version BuildPluginsVersion.KOTLIN
+    id("java-gradle-plugin")
+    id("maven-publish")
```

Comment:
what's about using https://github.com/vanniktech/gradle-maven-publish-plugin
the benefit is, it handles sources, javadoc, signing and the usage of `closeAndReleaseRepository` so we don't have to click buttons